internal/util/util_test.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package util contains some utility functions
+package util
+
+import (
+	"errors"
+	"io"
+	"syscall"
+	"testing"
+)
+
+func TestIsDisconnection(t *testing.T) {
+	cases := []struct {
+		err error
+		expected bool
+	}{
+		{nil, false},
+		{errors.New("foobar"), false},
+		{io.EOF, true},
+		{syscall.EPIPE, true},
+		{syscall.ETIMEDOUT, true},
```

Comment:
do we want to test `syscall.ECONNRESET` as `IsDisconnection` function test as well?

Reply:
Done